scripts/install

```diff
@@ -49,7 +49,7 @@ ynh_script_progression --message="Increasing maximum map count (sysctl)..."
 # Increase the maximum number of files inotify can monitor.
 cp -a ../conf/90-max_map_count-elasticsearch.conf /etc/sysctl.d/
 # Then, reload the kernel configuration.
-if ! [ "${container:-}" = "lxc" ] # lxc doesn't allow sysctl to play with kernel options.
+if ! [ "$(systemd-detect-virt)" == "lxc" ] # lxc doesn't allow sysctl to play with kernel options.
```

Comment:
Are you sure that it works? I don't see the `==` operator in `man test` page.

Otherwise, I propose this change:
```suggestion
if [ "$(systemd-detect-virt)" != "lxc" ] # lxc doesn't allow sysctl to play with kernel options.
```

Which improves the readability.

Reply:
Well, it worked for me, and I just reused what's been done in the mastodon app [here](https://github.com/YunoHost-Apps/mastodon_ynh/blob/c3c4416b24aaec57b8ec04f7479be84033259134/scripts/ynh_add_swap#L23)

Reply:
Right, I see that indeed both `=` and `==` are equivalent:
https://stackoverflow.com/questions/20449543/shell-equality-operators-eq